packages/components/src/Label/Label.spec.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react"
+import { render } from "@testing-library/react"
+import { Label, LabelProps } from "./Label"
+
+const defaultLabelProps = {
+  id: "someLabelId",
+  labelText: "Some field label",
+}
+
+const renderLabel = (props?: LabelProps): ReturnType<typeof render> => {
+  const mergedLabelProps = { ...defaultLabelProps, ...props }
+
+  return render(<Label {...mergedLabelProps} />)
+}
+
+describe("<Label />", () => {
+  it("renders a label", () => {
+    const { queryByText } = renderLabel()
+    expect(queryByText(defaultLabelProps.labelText)).toBeTruthy()
+  })
+
+  it("renders a `for` attribute", () => {
+    const htmlFor = "someFieldId"
+
+    const { container } = renderLabel({ htmlFor })
+    expect(container.querySelector(`[for="${htmlFor}"]`)).toBeTruthy()
+  })
+
+  it("renders an `id` attribute", () => {
+    const { container } = renderLabel()
+    expect(
+      container.querySelector(`[id="${defaultLabelProps.id}"]`)
+    ).toBeTruthy()
+  })
+
+  it("renders a reversed label", () => {
+    const { container } = renderLabel({ reversed: true })
+    expect(container.querySelector(".reversed")).toBeTruthy()
+  })
+
+  describe("label type", () => {
```

Comment:
Just delete this file and add the test cases to the stickersheet instead. These are either meant to be visual or are just testing that we spread the rest of the attributes into the component.